conbench/metrics.py

```diff
@@ -82,6 +82,14 @@ def decorate_flask_app_with_metrics(app) -> None:
 
     This mutates `app` in-place.
     """
+    # If the environment tells us something unique about us when we are
+    # one replica of N replicas, then inject the uniqueness too all emitted
+    # metrics -- see https://github.com/conbench/conbench/issues/1008
+    default_labels = {}
```

Comment:
Not: keep this balanced, as this blows up cardinatlity again.

kube-prometheus is doing this magic already; I was thinking that _one_ additional label is fine though for scrapers that are independent of kube-prometheus.